### PR TITLE
DOC-6763 - Copyright notice sweep

### PR DESCRIPTION
**Pull request by x8a**
Edited Copyright notices for consistency.

### CODE REVIEW

File: src/content/docs/licenses/product-or-service-licenses/mobile-app-licenses/android-application-licenses.mdx [hunk 1]
```diff
@@ -40,7 +40,7 @@ We love open-source software, and use the following in the New Relic Android app
       </td>
 
       <td>
-        Copyright 2011, Artem Votincev (apmem.org)
+        Copyright © 2011, Artem Votincev (apmem.org)
```
Comment (urbiz-nr) [inline src/content/docs/licenses/product-or-service-licenses/mobile-app-licenses/android-application-licenses.mdx:43]: I would remove Copyright, like in the instances above. Same goes for the rest of files.

**Review by urbiz-nr — CHANGES_REQUESTED** [review]
See my comment!

File: src/content/docs/licenses/product-or-service-licenses/mobile-app-licenses/android-application-licenses.mdx (hunk 1 above)
Comment (x8a) [reply src/content/docs/licenses/product-or-service-licenses/mobile-app-licenses/android-application-licenses.mdx:43]: According to what Joseph told us (https://newrelic.atlassian.net/browse/DOC-6763), for third part libraries, we should keep `Copyright ©`.

**Review by urbiz-nr — DISMISSED** [review]

**Comment by urbiz-nr** [comment]
@x8a you may merge!

File: src/content/docs/licenses/product-or-service-licenses/mobile-app-licenses/android-application-licenses.mdx (hunk 1 above)
Comment (jjang-nr) [reply src/content/docs/licenses/product-or-service-licenses/mobile-app-licenses/android-application-licenses.mdx:43]: They're technically the same, but with respect to New Relic's use of third-party code, I would rather dot every "i" as stated in the license terms

File: src/content/docs/licenses/product-or-service-licenses/new-relic-synthetics/new-relic-synthetics-licenses.mdx [hunk 2]
```diff
@@ -2611,7 +2611,7 @@ We love open-source software, and use the following in New Relic Synthetics. Tha
       </td>
 
       <td>
-        Copyright © 2011 - 2014 fengmk2 and other contributors. Copyright(c) 2015 - present node-modules and other contributors.
+        Copyright © 2011 - 2014 fengmk2 and other contributors. Copyright  2015 - present node-modules and other contributors.
```
Comment (jjang-nr) [inline src/content/docs/licenses/product-or-service-licenses/new-relic-synthetics/new-relic-synthetics-licenses.mdx:2614]: We need another copyright symbol here

**Review by jjang-nr — DISMISSED** [review]
Note added for a copyright symbol for one of the synthetics licenses. Approved with addition.

File: src/content/docs/licenses/product-or-service-licenses/new-relic-synthetics/new-relic-synthetics-licenses.mdx (hunk 2 above)
Comment (x8a) [reply src/content/docs/licenses/product-or-service-licenses/new-relic-synthetics/new-relic-synthetics-licenses.mdx]: Thanks for spotting it!

**Review by jjang-nr — APPROVED** [review]

**Review by jjang-nr — APPROVED** [review]

**Review by urbiz-nr — APPROVED** [review]
Looks good!

**Pull request merged**